src/meshdb/settings.py

```diff
@@ -213,3 +215,38 @@
 }
 
 HOOK_CUSTOM_MODEL = "meshapi_hooks.CelerySerializerHook"
+
+SPECTACULAR_SETTINGS = {
+    "TITLE": "MeshDB Data API",
+    "VERSION": "1.0.0",
+    "SERVE_INCLUDE_SCHEMA": False,
+    "TAGS": [
+        {"name": "API Status", "description": "Meta-endpoint to indicate API status"},
+        {"name": "Members", "description": "Members of the mesh and their contact details"},
+        {"name": "Buildings", "description": "Buildings on the mesh and their location data"},
+        {
+            "name": "Installs",
+            "description": "Installs, one corresponding to each household on the mesh or wishing to join",
+        },
+        {"name": "Links", "description": "Links between buildings"},
+        {"name": "Sectors", "description": 'Special antennas with broad coverage of a radial "slice" of land area'},
```

Comment:
Perhaps we could link to some documentation here? I don't think the concept of sectors is intuitive for non-meshers or those not familiar with RF.

Reply:
Our docs on this are quite thin but sure, linked this page https://docs.nycmesh.net/hardware/liteap/